notebooks/source/tbip.ipynb

```diff
@@ -0,0 +1,802 @@
+{
```

Comment:
<p>Please use NumPyro or numpyro rather than Numpyro through the tutorial.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyro-ppl/numpyro/pull/1498/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>